Installation instructions / help

Hi,

Thanks for sharing this tool!  I am trying to build nsntrace on Ubuntu 14.04.  After installing various required packages (libpcap, xsltproc, libnl-route-3) I got as far as `make` which fails with:

```
$ make
make  all-recursive
make[1]: Entering directory `/home/quux00/lang/c/nsntrace'
Making all in src
make[2]: Entering directory `/home/quux00/lang/c/nsntrace/src'
gcc -DHAVE_CONFIG_H -I. -I..  -I/usr/include/libnl3   -Wall -Wno-missing-braces   -g -O2 -MT nsntrace-net.o -MD -MP -MF .deps/nsntrace-net.Tpo -c -o nsntrace-net.o `test -f 'net.c' || echo './'`net.c
net.c:28:37: fatal error: netlink/route/link/veth.h: No such file or directory
 #include <netlink/route/link/veth.h>
```

The installation steps are not documented, but here's what I figured out:

```
$ ./autogen.sh
$ ./configure
(install any missing libs found)
$ make
```

Here's the output of my `configure` step:

```
$ ./configure 
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for style of include used by make... GNU
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for pcap_create in -lpcap... yes
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking pcap/pcap.h usability... yes
checking pcap/pcap.h presence... yes
checking for pcap/pcap.h... yes
checking for xsltproc... /usr/bin/xsltproc
checking for iptables... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for LIBNL... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating man/Makefile
config.status: creating tests/Makefile
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
```

I tried googling the `make` issue, but came up short. Thanks for any help/instructions you can provide

--- a/README.md
+++ b/README.md
@@ -98,3 +98,27 @@ $ sudo nsntrace -f tcp -o - wget www.google.com  2> /dev/null | tshark -r -
 13   0.113167 142.250.74.36 → 172.16.42.255 TCP 66 80 → 51088 [FIN, ACK] Seq=13815 Ack=143 Win=66816 Len=0 TSval=2846449554 TSecr=1362504583
 14   0.113190 172.16.42.255 → 142.250.74.36 TCP 66 51088 → 80 [ACK] Seq=143 Ack=13816 Win=64128 Len=0 TSval=1362504595 TSecr=2846449554
 ```
+
+# building from source
+To build nsntrace from source the following steps are needed:
+
+```
+$ ./autogen.sh
+$ ./configure
+$ make
+```
+And to install:
+```
+$ sudo make install
+```
+
+## dependencies
+The packages needed to build nsntrace are (Debian/Ubuntu style naming):
+* automake
+* docbook-xml
+* docbook-xsl
+* iptables
+* libnl-route-3-dev
+* lippcap-dev
+* pkg-config
+* xsltproc

